backend/spec/models/comment_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: comments
+#
+#  id                                  :bigint           not null, primary key
+#  content(コメントの内容)             :text(65535)      not null
+#  created_at                          :datetime         not null
+#  updated_at                          :datetime         not null
+#  memo_id(コメントが参照するメモのID) :integer          not null
+#
+# Indexes
+#
+#  index_comments_on_memo_id  (memo_id)
+#
+RSpec.describe Comment, type: :model do
+  subject(:comment) { build(:comment) }
+
+  describe 'バリデーションのテスト' do
+    context 'memo_idとcontentが有効な場合' do
+      let(:comment) { create(:comment) }
```

Comment:
FYI

以下のようにすると解消できますよ〜
こちらの方法だと、memosテーブルのレコードを保存するだけで良くなるので、
`create(:comment)`の場合のmemosテーブルとcommentsテーブルの保存より若干テストの実行速度が速くなります〜

```suggestion
      let(:memo) { create(:memo) }
      let(:comment) { build(:comment, memo: memo) }
```